scripts/tests/relayed-curl-portal-restart.sh

```diff
@@ -7,10 +7,6 @@ source "./scripts/tests/lib.sh"
 install_iptables_drop_rules
 trap remove_iptables_drop_rules EXIT # Cleanup after us
 
-client_curl_resource
```

Comment:
Yes, it tests that the client can open a new data channel if it's connected to a portal that just restarted. Otherwise, given the startup order, the client will create a data channel with a portal that started before it.

Reply:
Hm, but I see your point. Real-world clients will probably have some data channel already open. I'll revert.

Reply:
Updated to test both scenarios by restarting both with and without a binding